base/setup/welcome/CMakeLists.txt

```diff
@@ -4,4 +4,24 @@ add_rc_deps(welcome.rc ${welcome_rc_deps})
 add_executable(welcome welcome.c welcome.rc)
 set_module_type(welcome win32gui UNICODE)
 add_importlibs(welcome gdi32 user32 shell32 msvcrt kernel32 ntdll)
+
+# Reduce the required subsystem to WinNT 4.0 for i386 builds only.
```

Comment:
Out of curiosity: why?

Reply:
I think the commit message explains it :p

Reply:
Oh, also I need to get rid of the "LOCATION" property ^^

Reply:
Who on earth would use such an obscure feature, who on earth will implement such a thing and who on earth will maintain it? Trying to be realistic wouldn't hurt.

Reply:
Also @HBelusca have you tested it? I.e. run it on Windows NT 4 / Windows 2000? I seriously doubt it will work on NT4 anyway, e.g. it uses GetPrivateProfileSectionNames, which requires Windows 2000.

Reply:
![welcome_on_winnt4](https://user-images.githubusercontent.com/1969829/46620363-cb028000-cb24-11e8-94fe-d3c936802dda.png)